lib/app/features/protect_account/components/delete_twofa_input_step.dart

```diff
@@ -129,10 +129,12 @@ class DeleteTwoFAInputStep extends HookConsumerWidget {
       }
 
       if (next.hasError) {
-        showSimpleBottomSheet<void>(
-          context: ref.context,
-          child: const TwoFaTryAgainPage(),
-        );
+        WidgetsBinding.instance.addPostFrameCallback((_) {
```

Comment:
The problem is the order of listeners.
If we leave it the way it is, without the callback, this page is the first responder to the error event from the provider, causing the error modal route to be pushed. Right after that the listener from `passkeyDialog` is invoked, calling a `Navigator.pop` method, which doesn't pop the passkey dialog, but the new error modal.
It works fine for most other cases, because in addition to state listener for errors, we also have `ref.watch` used in the build method, which causes the error listener to reinitialize whenever the state changes and be moved at the end of the list of listeners for that provider. This changes the order of execution and makes it so that the passkey dialog is popped first and then the erorr modal is shown. It work exactly opposite in that particular case, because we don't have `ref.watch` anywhere, because we don't need it. Therefore a `postFrameCallback` is needed.